osu.Game/Screens/Play/PlayerConfiguration.cs

```diff
@@ -24,5 +24,10 @@ public class PlayerConfiguration
         /// Whether the player should be allowed to skip the intro, advancing to the start of gameplay.
         /// </summary>
         public bool AllowSkippingIntro { get; set; } = true;
+
+        /// <summary>
+        /// Whether the player should be allowed to skip the outro, advancing to the end of a storyboard.
+        /// </summary>
+        public bool AllowSkippingOutro { get; set; } = true;
```

Comment:
I would just bundle them all in one `AllowSkipping`, the point is whether or not the player is allowed to skip in time if they desire.